django/http/cookie.py

```diff
@@ -3,6 +3,10 @@
 # For backwards compatibility in Django 2.1.
 SimpleCookie = cookies.SimpleCookie
 
+# Monkey patching the Morsel object to support SameSite attribute added in 3.8
```

Comment:
`# Add support for the SameSite attribute (obsolete when PY37 is unsupported).`
I think the link to the PR isn't needed.